sdk/communication/azure-communication-identity/azure/communication/identity/__init__.py

```diff
@@ -41,11 +45,11 @@
 
 def __getattr__(name):
     if name == 'MicrosoftBotProperties':
-        warnings.warn(f"{name} is deprecated and should not be used.", DeprecationWarning)
-        from ._shared.models  import _MicrosoftBotProperties
-        return _MicrosoftBotProperties
```

Comment:
I didn't think the BotIdentifier was ever GA'ed in Identity?

Reply:
Removed the deprecation message.